packages/fe-fpm-writer/src/page/list.ts

```diff
@@ -12,9 +13,14 @@ import type { ListReport, InternalListReport } from './types';
  */
 function enhanceData(data: ListReport, manifest: Manifest): InternalListReport {
     const config: InternalListReport = { settings: {}, ...data, name: 'ListReport', ...getFclConfig(manifest) };
+    const minVersion = coerce(data.minUI5Version);
 
     // move settings into correct possition in the manifest
-    config.settings.entitySet = data.entity;
+    if (minVersion && minVersion.minor >= 94) {
```

Comment:
@schreckstefan, shouldn't we also check the major version of UI5? Just because there is no version higher than 1 does not really mean there never can be.

Reply:
Hi @Klaus-Keller you are basically right - I just had used the same logic as in other places of FPM writer. 
Then we should adjust them all.